unabridged-works-volume-3/original/15-letters.adoc

```diff
@@ -4435,15 +4647,20 @@ that thou mayst sing songs of degrees to the Redeemer of Israel,
 and mayest daily more and more partake of and rejoice in him, who is our joy,
 and the crown thereof.
 
+[.signed-section-closing]
 Thy friend, in the most sincere, tender love,
 
+[.signed-section-signature]
 I+++.+++ P.
 
+[.signed-section-context-close]
 11th of Fifth Month, 1677
 
-TO THOMAS WALMSLEY
+[.letter-heading]
+To Thomas Walmsley
 
-FRIEND T. W.,
+[.salutation]
+Friend T.W.,
```

Comment:
So I wasn't sure what to do about this. And it seemed to be fine in another letter near the beginning of the chapter so I wasn't sure why it didn't work here. Is it because there wasn't a "Dear" in the beginning?

Reply:
It's because of the lack of space between the initials, (I know, not super clear from the error). But if you add back a space before the `W` it will be OK.

Reply:
ahhh got it!